ietf/static_src/css/iab.scss

```diff
@@ -171,3 +181,34 @@
 .iab_body {
     background-color: rgba(62, 139, 193, 0.08);
 }
+
+.dropdown-toggle__iab {
+    &:after {
+        display: inline;
+        vertical-align: -0.75rem;
+    }
+}
+
+.iab-tabs > li {
+    font-size: rem(16);
+    line-height: 1;
+}
+
+.iab-tabs > li > a {
+    padding: rem(16) rem(18) rem(10);
+    text-align: left;
+}
+
+.iab-tabs > li > .active {
+    color: var(--color-iab-navy) !important;
+    border-bottom: 2px solid black !important;
+}
+
+.iab-tabs > li > :not(.active) {
+    color: #616466 !important;
+    border-bottom: 2px solid var(--color-iab-grey-5) !important;
+
+    &:hover {
+        color: var(--color-iab-navy) !important;
+    }
+}
```

Comment:
Beware the specificity creep here. I see there's even more `!important`s being added. With BEM usually we can rely on a flat specificity graph (everything stylable has a class) and source order, where modifiers are declared after the base class.

Should you do e.g. 
- `<li class="iab-tabs__item">`
- `<a href="..." class="iab-tabs__link">`
- `<a href="..." class="iab-tabs__link active">` (or `<a hred="..." class="iab-tabs__link "iab-tabs__link--active">`)

Something to think about if, you haven't already. I know this project uses bootstrap (i.e. `active` may be baked in), so we can't necessarily do the typical BEM thing..

Reply:
Yep, trying to customise something bootstrap didn't really contemplate customising lead to more `!important`s than I wanted. I've given it another going over and removed some that didn't need to be there, but yeah, I don't think we can do perfect BEM and keep the client's requirement for bootstrap in this case.